VideoLocker/build.gradle

```diff
@@ -89,17 +89,19 @@ class Configuration {
         // is undocumented, but is necessary because it tries to read
         // the meta-data at compile time so resource references don't work
         def fabric = config.get('FABRIC');
-        def fabric_key = fabric.get('FABRIC_KEY') ?: ""
-        def fabric_secret = fabric.get('FABRIC_BUILD_SECRET')
-        if(fabric_key != null && fabric_secret != null) {
-            def crashlyticsPath = projectDir + '/crashlytics.properties'
-            def writer = new FileWriter(crashlyticsPath)
-            writer.write(
-"""
+        if (fabric != null) {
+            def fabric_key = fabric.get('FABRIC_KEY') ?: ""
```

Comment:
I'm not sure we we need the ``?:`` any more. Since we just check if it's ``null`` right below.